wrappers/python/simtk/openmm/app/internal/pdbstructure.py

```diff
@@ -795,11 +798,14 @@ def __init__(self, pdb_line, pdbstructure=None):
         try: self.formal_charge = int(pdb_line[78:80])
         except ValueError: self.formal_charge = None
         # figure out atom element
-        try:
-            # Try to find a sensible element symbol from columns 76-77
-            self.element = element.get_by_symbol(self.element_symbol)
-        except KeyError:
-            self.element = None
+        if self.element_symbol == extraParticleIdentifier:
+            self.element = 'EP'
```

Comment:
I thought we wanted to set `element = None` in this case, since `atom.element` is either an `Element` object or `None` (indicating a dummy particle).  Allowing it to be a string too seems weird.  Or am I misunderstanding how downstream processing works?